zellij-utils/src/input/actions.rs

```diff
@@ -12,6 +12,16 @@ pub enum Direction {
     Down,
 }
 
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub enum ResizeDirection {
```

Comment:
Not sure why would you need a separate enum for that here, given that you aren't really using `Direction` in the `Action` enum anymore?

Reply:
`Direction` enum is used in variants like `MoveFocus`, `MoveFocusOrTab` where `Increase` and `Decrease` variants don't make much sense, So I made a separate enum for that.